src/microfrontends/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.12.0",
+  "version": "1.13.0",
```

Looks like you'll need to bump this again. We're already at 1.14.0
```suggestion
  "version": "1.15.0",
```